luigi/scheduler.py

```diff
@@ -806,6 +564,7 @@ def _update_priority(self, task, prio, worker):
         If the task doesn't exist, a placeholder task is created to preserve priority when the task is later scheduled.
         """
         task.priority = prio = max(prio, task.priority)
+        self._state.persist_task(task)
```

Comment:
why do we need to persist?

Reply:
The line above, we set `task.priority`, so then we have to 💾 the change